app/gimlet/base.toml

```diff
@@ -110,7 +110,7 @@ notifications = ["i2c2-irq", "i2c3-irq", "i2c4-irq"]
 [tasks.spd]
 name = "task-gimlet-spd"
 features = ["h753"]
-priority = 2
+priority = 4
```

Comment:
Is this intentional?  `task-gimlet-spd` has traditionally had a high priority because it needs to reply to real-time events (I2C requests from the host).

Reply:
agh, nope, that was definitely not intentional.